src/main/tools/linux-sandbox-options.cc

```diff
@@ -234,7 +234,8 @@ static void ParseCommandLine(unique_ptr<vector<char *>> args) {
         break;
       case 'C':
         ValidateIsAbsolutePath(optarg, args->front(), static_cast<char>(c));
-        opt.cgroups_dir.assign(optarg);
+        opt.cgroups_dirs.emplace_back(optarg);
+        opt.writable_files.emplace_back(optarg);
```

Comment:
Why do we need to make the cgroups dir writable? We wouldn't want spawns to be able to write to the cgroups dir, circumventing any existing cgroup limits placed on the spawn by Bazel.

Reply:
Ah yes, this was removed recently in da8c0806da4ca0f4280f7b362359e484db22c405 which clamps down on the write access to the cgroup dir (previously the spawn could actually modify its own limits).